listenbrainz/webserver/static/js/src/utils/APIService.ts

```diff
@@ -572,6 +572,21 @@ export default class APIService {
     return response.json();
   };
 
+  getFeedbackForUserForRecordingsNew = async (
```

Comment:
Will remove getFeedbackForUserForRecordings and rename this one to that, once all usages are migrated.